solver/llbsolver/history.go

```diff
@@ -392,6 +516,10 @@ func (h *HistoryQueue) Listen(ctx context.Context, req *controlapi.BuildHistoryR
 		}
 	}
 
+	if req.NoWait {
+		return nil
+	}
```

Comment:
`NoWait` is if the client only wants to get a list of builds once and keep waiting on updates for them. Currently, the only use case is in the tests where because some tests check for cleaned contentstore the full build history also needs to be cleaned.

Reply:
Could we rename it to `Exit` or `EarlyExit` or similar?